.github/workflows/kuksa_val_docker_amd64.yml

```diff
@@ -0,0 +1,58 @@
+name: kuksa_val_docker amd64
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    
+jobs:
+  build:
+    runs-on: [ self-hosted ]
```

Comment:
Should mention the correct task, to prevent amd64 builds on wrong architecture (ALL private runners wil listen to self.hosted tag).

kuksa-x64 shoudl be used here